### PR TITLE
gittigidiyor.com

### PR DESCRIPTION
**Pull request by gork7777**

### CODE REVIEW

File: TurkishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74572
+@@||mcdn01.gittigidiyor.net/rw/media/*$domain=gittigidiyor.com
```
Comment (zloyden) [inline TurkishFilter/sections/whitelist.txt:3]: Hm, too wide rule.
1. You should add `~gittigidiyor.com` to the `/reklam-$domain=~avansas.com`.
2. Exchange your exception to `@@||mcdn*.gittigidiyor.net/*/media/images/*/ads-packages.css$domain=gittigidiyor.com`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/whitelist.txt (hunk 1 above)
Comment (gork7777) [reply TurkishFilter/sections/whitelist.txt:3]: But `/reklam-$domain=~avansas.com` rule is in this general_url file, are you sure to add in this?

Comment (zloyden) [reply TurkishFilter/sections/whitelist.txt:3]: Ofc, that rule is caused it.
After your changes about `1.` it should be `/reklam-$domain=~avansas.com|~gittigidiyor.com`

Comment (RadicalNoise) [reply TurkishFilter/sections/whitelist.txt:3]: `~` karakteri istisna demektir. Önüne eklenen alan adını beyaz listeye alır. @gork7777 

https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#basic-modifiers

Comment (zloyden) [reply TurkishFilter/sections/whitelist.txt]: Don't worry, I made the necessary changes.

**Review by zloyden — APPROVED** [review]

**Pull request merged**